### PR TITLE
Make the "Help" and "Print" buttons *nix (Linux and OSX) compatible.

### PR DESCRIPTION
**Pull request by gutow**
Small additions to the code so that the .pdf files will open properly on Linux and OSX.

Addresses issue #72.

### CODE REVIEW

File: src/user_interface.py [hunk 1]
```diff
@@ -610,19 +618,28 @@ def __confirm(self):
...
+                subprocess.Popen([helpfile], shell=True)
         elif (self.form_variant == FormVariantSelection.VARIANT_150_Q):
             helpfile = str(
                 Path(__file__).parent / "assets" /
                 "multiple_choice_sheet_150q.pdf")
-            subprocess.Popen([helpfile], shell=True)
+            if platform.system() in ('Darwin','Linux'):
+                subprocess.Popen(['open', helpfile])
+            else:
+                subprocess.Popen([helpfile], shell=True)
```
Comment (iansan5653) [inline src/user_interface.py:642]: Could you please remove all the other changes from this pull request and only include these three changes? That way I can just merge this fix and we can address the scrolling changes in https://github.com/iansan5653/open-mcr/pull/74.

**Review by iansan5653 — CHANGES_REQUESTED** [review]

File: src/user_interface.py (hunk 1 above)
Comment (gutow) [reply src/user_interface.py:642]: I will cherry pick as soon as I get a chance. We are desperate here. So, I am focused on getting something we can use first.

Comment (gutow) [reply src/user_interface.py:634]: Unless I missed something the commits below resolve this by removing  the scrolling commits.

**Review by iansan5653 — APPROVED** [review]
Looks great! I tested on my MacBook and it works well.

**Pull request merged**